src/pages/auth/LoginPage.jsx

```diff
@@ -0,0 +1,12 @@
+const { Login } = require("components/auth");
+const { Navbar } = require("components/navigation");
+
+const LoginPage = () => {
+  return (
+    <>
+      <Navbar />
```

Comment:
Instead of putting the navbar on every page, you can put it in App.js

Reply:
Do you think it's good from performance point of view? Or is it to avoid code repetition?

Reply:
I am saying it from a reusability point of view. Why did you put it on every page? while you can simply put it in app.js bcz that's how we usually do it in react.

Reply:
Makes sense. Will refactor. Thanks!